src/deluge/storage/DX7Cartridge.h

```diff
@@ -87,31 +88,31 @@ class DX7Cartridge {
 	 * Returns EMPTY_STRING if it was parsed successfully
 	 * otherwise a string describing the error.
 	 */
-	deluge::l10n::String load(const uint8_t* stream, size_t size) {
+	deluge::l10n::String load(std::span<std::byte> stream) {
 		using deluge::l10n::String;
-		const uint8_t* pos = stream;
+		const std::byte* pos = stream.data();
 
 		size_t minMsgSize = 163;
 
-		if (size < minMsgSize) {
-			memcpy(voiceData + 6, pos, size);
+		if (stream.size() < minMsgSize) {
+			memcpy(voiceData + 6, pos, stream.size());
 			return String::STRING_FOR_DX_ERROR_FILE_TOO_SMALL;
 		}
 
-		if (pos[0] != 0xF0) {
+		if (pos[0] != std::byte{0xF0}) {
 			// it is not, just copy the first 4096 bytes
 			memcpy(voiceData + 6, pos, 4096);
 			return String::STRING_FOR_DX_ERROR_NO_SYSEX_START;
 		}
 
 		int i;
```

Comment:
`i` should be `size_t` (or `std::span<std::byte>::size_type` if you want to be verbose about it)